readme.md

```diff
@@ -163,6 +163,19 @@ When a server comes online, it will have no initial knowledge of clients connect
 ```
 All other servers respond by sending `client_update`
 
+#### Server Hello 
+When a server added another server's IP to its list of connected servers. It sends this message to that server. To Let them know that this server is connecting to them. 
+```JSON
+{
+"type": "server_hello"
+   "data" {

```

Comment:
Since no 'hello' message was exchanged, how would we know what public key was used to sign this data?